### ISSUE TITLE
In scikit-learn wrapper, to caclulate AUC, should I use predict_proba?

### ISSUE THREAD

**Issue by johnny5550822** [dmlc/xgboost#1961]
I want to use xgboost through scikit-learn wrapper. Let say I fit a model and I want to calculate the AUC. Should I use .predict_proba? (since there is no .decision_function). As far as I know, for small dataset, .predict_proba is not proper for SVM because random prob. will be generated and therefore I need to use .decision_fucntion for SVM. What about scikit-learn?

e.g., of using the xgboost with scikit-learn:
`score_ = model_best.fit(features[train_i,:],labels[train_i]).predict_proba(features[test_i,:]) # for xgboost`

**Comment by mlgruby** [comment]
Yes for scikit-learn wrapper use .predict_proba to get y-predicted then you can use sklearn.metrics.auc method.
